lib/general_widgets/custom_user_channel.dart

```diff
@@ -2,37 +2,43 @@ import 'package:flutter/material.dart';
 
 class CustomUserChannel extends StatelessWidget {
   const CustomUserChannel(
-      {Key? key, this.image, this.text, this.trailingIcon, this.iconColor})
+      {Key? key, this.image, this.text, this.text2, this.trailingIcon, this.iconColor})
       : super(key: key);
   final image;
   final text;
+  final text2;
   final trailingIcon;
   final iconColor;
 
   @override
   Widget build(BuildContext context) {
     return Container(
-        height: 40,
-        width: 379,
+        height: 24,
+        width: 211,
         child: Row(
           children: [
             Container(
               child: Image(
-                height: 34,
-                width: 34,
+                height: 24,
```

Comment:
Same here

Reply:
same screen-utils package used to access some properties that can't be accessed using the const textStyle

Reply:
same screen-utils package used to access some properties that can't be accessed using the const textStyle